formats/MrssFormat.php

```diff
@@ -78,7 +78,9 @@ public function stringify(){
 		}
 
 		$charset = $this->getCharset();
-
+		
+		/* xml attributes need to have certain characters escaped to be w3c compliant */
+		$titleSafe = htmlspecialchars($title);
```

Comment:
The variable name should reflect its use, like `imageTitle`. Additionally, make sure to only encode double quotes, so other items don't get encoded as well (like "<" turning into "&lt"). You can use the [`ENT_COMPAT`](http://php.net/manual/en/function.htmlspecialchars.php) flag for that:

```PHP
$imageTitle = htmlspecialchars($title, ENT_COMPAT);
```

>Will convert double-quotes and leave single-quotes alone.
>
>-- http://php.net/manual/en/function.htmlspecialchars.php

Reply:
Travis currently [complains](https://travis-ci.org/RSS-Bridge/rss-bridge/jobs/426200751) about "Whitespace found at end of line", please make sure to remove whitespace from the end of lines.